### PR TITLE
chore: update blue logo for updated molgenis-theme

### PR DESCRIPTION
**Pull request by jvanveen**
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] Migration step added in case of breaking change
- [ ] User documentation updated
- [ ] (If you have changed REST API interface) view-swagger.ftl updated
- [ ] Test plan template updated
- [ ] Clean commits
- [ ] Added Feature/Fix to release notes

### CODE REVIEW

**Comment by fdlk** [comment]
The logo belongs in the css to begin with. I realize we have all sorts of settings and stuff but perhaps we can start by moving the default logos to css?

**Review by sidohaakma — COMMENTED** [review]
The first one needs to be empty? ;-D. Is this copied from the themes repo?

File: molgenis-core-ui/src/main/resources/img/Logo_Blue_Small.svg [hunk 1]
```diff
@@ -1,40 +1,158 @@
...
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="132px"
+   height="43px"
+   viewBox="0 0 132 43"
+   version="1.1"
+   id="svg1017"
+   sodipodi:docname="Logo_Blue_Small.svg"
+   inkscape:export-filename="/home/jeroen/code/molgenis/molgenis-core-ui/src/main/resources/img/Logo_Blue_Small.png"
```
Comment (jelmerveen) [inline molgenis-core-ui/src/main/resources/img/Logo_Blue_Small.svg:16]: do we need inkscape specific code?

File: molgenis-core-ui/src/main/resources/img/Logo_Blue_Small.svg [hunk 2]
```diff
@@ -1,40 +1,158 @@
...
+  </metadata>
+  <sodipodi:namedview
+     pagecolor="#ffffff"
+     bordercolor="#666666"
+     borderopacity="1"
+     objecttolerance="10"
+     gridtolerance="10"
+     guidetolerance="10"
+     inkscape:pageopacity="0"
+     inkscape:pageshadow="2"
```
Comment (jelmerveen) [inline molgenis-core-ui/src/main/resources/img/Logo_Blue_Small.svg:40]: same here :)

File: molgenis-core-ui/src/main/resources/img/Logo_Blue_Small.svg (hunk 1 above)
Comment (jvanveen) [reply molgenis-core-ui/src/main/resources/img/Logo_Blue_Small.svg:16]: Source file was edited with inkscape

**Pull request merged**